tests/test_LogOutputHelper.cpp

```diff
@@ -257,14 +257,15 @@ BOOST_AUTO_TEST_CASE(Fip)
     Opm::LogOutputHelper<double> helper(eclState, schedule, st);
     Opm::Inplace initial, current;
     const auto& phases = current.phases();
+    int offset = 17;
```

Comment:
phases.size() = 17 before this PR. It is only used to set different inplace values. Any number would do, as long as it matches the reference numbers. So instead of changing the reference values I just freeze the values set in the test.